types/prop-types/prop-types-tests.ts

```diff
@@ -48,6 +48,10 @@ const arrayOfTypes = [PropTypes.string, PropTypes.bool, PropTypes.shape({
 })];
 type PropTypesMap = PropTypes.ValidationMap<Props>;
 
+const customPropFunc: PropTypes.Validator<typeof uniqueType> = () => null;
```

Comment:
We write a lot of custom prop functions like this, so thought I would include that here as well.

Side note: This is ugly and we can't figure out a better pattern.